node/task.ts

```diff
@@ -812,6 +812,62 @@ export interface FindOptions {
     followSymbolicLinks: boolean;
 }
 
+/**
+ * Defines if path is unc-path.
+ *
+ * @param path  a path to a file.
+ * @returns     true if path starts with double backslash, otherwise returns false.
+ */
+export function isUncPath(path: string) {
+    return /^\\\\[^\\]/.test(path);
+}
```

Comment:
Could it make sense to move this function to [internal.ts](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/internal.ts)?